src/Component/Symbolizer/Editor/Editor.tsx

```diff
@@ -75,6 +79,8 @@ export interface EditorComposableProps {
   rasterEditor?: {
     visibility?: boolean;
   };
+  // TODO support default value for visibilityField
```

Comment:
Do we really need a configurable default value for this? IMHO we don't need this TODO here.

Reply:
I do not see a clear use-case here, but why restrict this field while the others do have support for this?